### PR TITLE
feature: [VMM-584] New monitoring notebook with batch reference data example

### PR DESCRIPTION
**Pull request by ewagner-verta**
## Impact and Context
Adds a new example notebook to the monitoring directory, which provides a working example of batch reference data ingestion and produces meaningful data in the charts and graphs for monitoring drift etc.

## Risks and Area of Effect
Does not affect production code.  The example notebook has been validated in several environments.

## Testing
- [ ] Unit test
- [X] Deployed to dev env
- [ ] Other (explain) 

## Reverting
- [ ] Contains Migration - _Do Not Revert_

### CODE REVIEW

**Comment by ewagner-verta** [comment]
Waiting to merge this example until after the next release as it relies on changes contained therein.

**Comment by conradoverta** [comment]
We should call teh notebook `isOdd` for clarity. I didn't realize it was the `isOdd` until I read it

File: monitoring/Monitoring-batch-reference-data-example.ipynb [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Use local env vars or uncomment and fill out the lines below:\n",
+    "os.environ['VERTA_EMAIL'] = 'ewagner@verta.ai'\n",
+    "os.environ['VERTA_DEV_KEY'] = 'fb0c7d6c-8f91-4f87-8730-febd36f74553'\n",
+    "os.environ['VERTA_HOST'] = 'ewagner2.dev.verta.ai'\n",
```
Comment (conradoverta) [inline monitoring/Monitoring-batch-reference-data-example.ipynb:92]: never commit these

File: monitoring/Monitoring-batch-reference-data-example.ipynb [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Use local env vars or uncomment and fill out the lines below:\n",
+    "# os.environ['VERTA_EMAIL'] = ''\n",
+    "# os.environ['VERTA_DEV_KEY'] = ''\n",
+    "# os.environ['VERTA_HOST'] = ''\n",
+    "\n",
+    "NAME: str = \"batch_reference_data_example-old\"\n",
```
[the remark this answers is not in the document]
Comment (conradoverta) [reply monitoring/Monitoring-batch-reference-data-example.ipynb]: old comment. not sure if still relevant

File: monitoring/Monitoring-batch-reference-data-example.ipynb [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+    "                \"isOdd\": is_odd,\n",
+    "                \"isOdd.confidence\": 1.0,\n",
+    "            }\n",
+    "        )\n",
+    "    return pd.DataFrame(data)\n",
+    "\n",
+    "\n",
+    "df_reference = generate_day_data(batch_size=1000)\n",
+    "X_reference = df_reference[[\"id\"]]\n",
+    "y_reference = df_reference[\"isOdd\"]"
```
[the remark this answers is not in the document]
Comment (conradoverta) [reply monitoring/Monitoring-batch-reference-data-example.ipynb]: old comment. not sure if still relevant

File: monitoring/Monitoring-batch-reference-data-example.ipynb (hunk 2 above)
[the remark this answers is not in the document]
Comment (ewagner-verta) [reply monitoring/Monitoring-batch-reference-data-example.ipynb]: Fixed to a generic name

File: monitoring/Monitoring-batch-reference-data-example.ipynb (hunk 1 above)
Comment (ewagner-verta) [reply monitoring/Monitoring-batch-reference-data-example.ipynb]: Forgot to strip that out before committing.  My bad.  I have removed that key and generated a new one.

**Review by coryverta — APPROVED** [review]

**Review by conradoverta — APPROVED** [review]

**Pull request merged**